baseballstadium/src/__tests__/dashboard.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { render, fireEvent } from 'react-testing-library';
+import 'jest-dom/extend-expect';
+import Dashboard from '../components/dashboard';
+import Display from '../components/display';
+
+describe('<Dashboard />', () => {
+  it('Strike Button Clicked', () => {
+    const strikes = 0;
+
+    const {
```

Comment:
This is really clean, I haven't seen anybody destructure this way yet. Very nice.